docs/content/docs/ops/metrics.md

```diff
@@ -1717,6 +1717,11 @@ Note that the metrics are only available via reporters.
       <td>Maximum recorded duration of a single consecutive period of the task being in the hard back pressure state in the last sampling period. Please check softBackPressuredTimeMsPerSecond and hardBackPressuredTimeMsPerSecond for more information.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td>ioBackPressuredTimeMsPerSecond</td>
+      <td>The time (in milliseconds) this task is back pressured by the uploading of Changelog state backend, only positive when Changelog state backend is enabled. Please check 'dstl.dfs.upload.max-in-flight' for more information.</td>
+      <td>Gauge</td>
```

Comment:
I think we should generalize this description a bit or change the name of the metric to something with `changelog` name.

Either instead of saying that this metric only reports changelog time, rename `ioBackPressuredTime` to `ioTime` (vide my other comment), describe it as "time spent by task doing IO operations, currently only takes into account changelog state backend."

or rename the metric to `changelogTime` or `changelogBusyTime`, and keep the very detailed description.

?